systemtest/070-list-tags.bats

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bats
+#
+# list-tags tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+
+    start_registry --enable-delete=true reg
+}
+
+# list from registry
+@test "list-tags: remove image from registry" {
+    local remote_image=docker://quay.io/libpod/busybox:latest
+    local img_tag=unsigned
+    local localimg=docker://localhost:5000/busybox:${img_tag}
+    local output=
+
+    run_skopeo copy --dest-tls-verify=false $remote_image $localimg
+    output=$(run_skopeo inspect --tls-verify=false --raw $localimg)
+    echo $output | grep "vnd.docker.distribution.manifest.v2+json"
+
+    output=$(run_skopeo list-tags --tls-verify=false docker://localhost:5000/busybox)
+    echo $output | grep "${img_tag}"
+}
+
+# list from a local docker-archive file
+@test "list-tags: from a docker-archive file" {
+    local file_name=docker-two-images.tar.xz
+    local output=
+
+    curl -o $file_name https://raw.githubusercontent.com/containers/common/main/libimage/testdata/docker-two-images.tar.xz
```

Comment:
Could you move the file here?  This way, the Skopeo tests are independent.

Reply:
> Could you move the file here? This way, the Skopeo tests are independent.

I will do this